tests/acceptance/features/apiActivities/activities.feature

```diff
@@ -2134,6 +2134,302 @@ Feature: check activities
       }
       """
 
+  @issue-10150
+  Scenario: check activity with limit and sort filter(asc|desc)
+    Given user "Alice" has created folder "/New Folder"
+    And user "Alice" has created folder "/New Folder/Sub Folder"
+    And user "Alice" has uploaded file with content "ownCloud test text file 0" to "/New Folder/Sub Folder/textfile.txt"
+    When user "Alice" lists the activities of folder "New Folder" from space "Personal" with limit "2" and sort "desc" using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
```

Comment:
The main thing here is to check the order of activities. And this step doesn't do that.
So see https://github.com/owncloud/ocis/pull/10235#discussion_r1789447753